JitsiConferenceEventManager.js

```diff
@@ -51,14 +48,6 @@ export default function JitsiConferenceEventManager(conference) {
                 track.isMuted(),
                 track.getType());
         });
-    conference.on(
-        JitsiConferenceEvents.CONNECTION_INTERRUPTED,
-        Statistics.sendEventToAll.bind(
-            Statistics, CONNECTION_INTERRUPTED));
-    conference.on(
-        JitsiConferenceEvents.CONNECTION_RESTORED,
-        Statistics.sendEventToAll.bind(
-            Statistics, CONNECTION_RESTORED));
```

Comment:
Why was this removed? It was firing at the wrong times?

Reply:
We don't need the CONNECTION_RESTORED analytics event, because we can use the ICE_STATE_CHANGED events instead of it